src/main/java/org/broadinstitute/hellbender/tools/sv/cluster/CanonicalSVCollapser.java

```diff
@@ -239,74 +268,85 @@ protected Allele collapseRefAlleles(final String contig, final int pos) {
+        } else {
+            alleles = collapseNonCNVGenotypeAlleles(genotypes, expectedCopyNumber, refAllele);
+        }
+        return alleles;
+    }
+
     /**
-     * Collapses a set of genotypes from a single sample into a list of alleles for a representative genotype. Ploidy
-     * is determined with the {@link #collapseExpectedCopyNumber(Collection)} method. The result is the most common non-ref genotype
-     * alleles list, potentially augmented with additional ref alleles to match the ploidy.
+     * Collapses a set of genotypes from a single sample into a list of alleles for a representative genotype. The
+     * result is the most common non-ref genotype alleles list, potentially augmented with additional ref alleles
+     * to match the ploidy.
      * @param genotypes genotypes from a single variant and single sample
      * @param refAllele ref allele for the genotype
-     * @param sampleAltAlleles unique alt alleles
      * @return list of alleles for a representative genotype (may be empty)
      */
-    @VisibleForTesting
-    protected List<Allele> collapseSampleGenotypeAlleles(final Collection<Genotype> genotypes,
-                                                         final int expectedCopyNumber,
-                                                         final Allele refAllele,
-                                                         final List<Allele> sampleAltAlleles) {
+    private List<Allele> collapseNonCNVGenotypeAlleles(final Collection<Genotype> genotypes,
+                                                       final int expectedCopyNumber,
+                                                       final Allele refAllele) {
         Utils.nonNull(genotypes);
-        Utils.nonEmpty(genotypes);
 
-        // No alt alleles, so return all ref
-        if (sampleAltAlleles.isEmpty()) {
-            return Collections.nCopies(expectedCopyNumber, refAllele);
+        // Trivial case
+        if (genotypes.size() == 1) {
+            return genotypes.iterator().next().getAlleles();
         }
 
-        // Ploidy 0
-        if (expectedCopyNumber == 0) {
+        // Empty case
+        if (genotypes.isEmpty()) {
             return Collections.emptyList();
         }
 
-        // Multi-allelic CNVs - genotypes are stored in copy number attribute
-        if (sampleAltAlleles.size() == 2 && (sampleAltAlleles.contains(Allele.SV_SIMPLE_DEL)
-                || sampleAltAlleles.contains(Allele.SV_SIMPLE_DUP))) {
-            return Collections.nCopies(expectedCopyNumber, Allele.NO_CALL);
-        }
-
-        // Cannot resolve alleles for DUPs when ploidy is over 1
-        if (expectedCopyNumber > 1 && sampleAltAlleles.size() == 1
-                && sampleAltAlleles.get(0).equals(Allele.SV_SIMPLE_DUP)) {
-            return Collections.nCopies(expectedCopyNumber, Allele.NO_CALL);
+        // Ploidy 0
+        if (expectedCopyNumber == 0) {
```

Comment:
Some of the issues you are talking about here in your second paragraph are behind some of the changes that are coming to the VCF spec in version 4.4. In particular, the `SVCLAIM` info field will allow you to specify whether your deletion and duplication calls are really claiming that there are breakpoints at the specified location, or if the claim is simply that the copy number of the segment on the reference differs from expectations based on read depth. My initial thought when reading this segment of code was that representing things in this style might allow us to preserve the calls we are throwing away here. What do you think of that approach?

Genome STRiP measures the copy number of repeated sequences like segmental duplications and represents it by adding a "duplicate segments" info field. So for example if there are two copies of a segmental duplication in the reference on the autosomes (so you expect a reference sample to be copy number 4), and the copy number is measured to be 5 but we don't know which side of the seg dup is duplicated, it writes a record at one locus with the duplicate segment info field pointing to the other segment and the sample's CN format field set to 5. In VCF 4.4 this sort of thing would still make sense using a depth SVCLAIM type representation.

Reply:
It would be very realistic to adopt 4.4 in the output in the near future (or have a setting to select which spec to use), and it seems like it would be easy to set SVCLAIM at the end of the pipeline using the EVIDENCE field. Carrying it through certainly has its advantages, including resolving cases like the one you suggested above, but might be hard to implement in gatk-sv given the current scope of the pipeline.